git/util.py

```diff
@@ -652,7 +652,7 @@ def _parse_progress_line(self, line: AnyStr) -> None:
             op_code |= self.CHECKING_OUT
         else:
             # Note: On Windows it can happen that partial lines are sent.
-            # Hence we get something like "CompreReceiving objects", which is
+            # Hence we get something like "CompareReceiving objects", which is
```

Comment:
This word is in quotes and should be left alone.

Reply:
```suggestion
            # Hence we get something like "CompreReceiving objects", which is
```